server/graphql/v2/query/collection/TagStatsCollectionQuery.js

```diff
@@ -0,0 +1,18 @@
+import { GraphQLNonNull } from 'graphql';
+import { pick } from 'lodash';
+
+import rawQueries from '../../../../lib/queries';
+import { TagStatsCollection } from '../../collection/TagStatsCollection';
+
+const TagStatsCollectionQuery = {
+  type: new GraphQLNonNull(TagStatsCollection),
+  async resolve(_, args) {
+    const tagFrequencies = await rawQueries.getTagFrequencies({
+      ...pick(args, ['limit', 'offset']),
+    });
+
+    return { nodes: tagFrequencies, limit: args.limit, offset: args.offset };
```

Comment:
I don't see any `args` defined here. Or am I missing something?

Reply:
Ops, correct I've missed adding this to v2. Anyways, I've migrated this and the [relevant front-end PR](https://github.com/opencollective/opencollective-frontend/pull/7611) to use v2 now so this is fixed. 👍🏽

Reply:
I still don't see any `args` being defined here

Reply:
Sorry about this. I was a bit confused on this one. Initially I didn't add an `arg` here since we were tag frequencies based on all the collectives. I've now added support to only return tags based on the `searchTerm`.

Reply:
I still don't see `limit` and `offset` in `args`

Reply:
Oh... 🤯 .... I've added the `limit` and `offset` as well here. 👍🏽